coral/src/app/pages/login/index.tsx

```diff
@@ -0,0 +1,43 @@
+import z from "zod";
```

Comment:
I think this page not not yet be accessed via browser. Could you add the `/login` route for this page? Also, should this file be `coral/src/app/pages/login.tsx` ? AFAIK the `index.tsx` should be only saved for resource roots. (So my `/hello` was probably a bit misleading filename)

Reply:
💡  will do!